docs/source/azure/advanced/azurefunctions.rst

```diff
@@ -36,12 +36,13 @@ Execution in Azure functions comes with a default set of configurations for the
 resources. To override these settings you must set 'provision-options' with one of the following
 keys:
 
+- groupName (default: cloud-custodian)
 - location (default: West US 2)
 - appInsightsLocation (default: West US 2)
-- servicePlanName (default: cloud-custodian)
-- sku (default: Basic)
-- skuCode (default: B1)
-- workerSize (default: 0)
+- servicePlanName (default: cloud-custodian-plan)
+- skuTier (default: Basic)
+- skuName (default: B1)
+- storageName (default: custodianstorageaccount)
```

Comment:
The default isn't accurate since storage account names have to be unique across azure.  Do we add a guid onto this if someone has already run this template and claimed custodianstorageaccount?